### PR TITLE
Add convet_to_tflite script for stable diffusion 1.5 model

### PR DESCRIPTION
**Pull request by yichunk**
Add convet_to_tflite script for stable diffusion 1.5 model
Download pytorch model weights from https://github.com/kjsman/stable-diffusion-pytorch

BUG=b/311216181

### CODE REVIEW

File: ai_edge_torch/generative/examples/stable_diffusion/convert_to_tflite.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  encoder.load_state_dict(torch.load(encoder_ckpt_path))
+
+  diffusion = Diffusion()
+  diffusion.load_state_dict(torch.load(diffusion_ckpt_path))
+
+  decoder = Decoder()
+  decoder.load_state_dict(torch.load(decoder_ckpt_path))
+
+  # Tensors used to trace the model graph during conversion.
+  n_tokens = 77
```
Comment (haozha111) [inline ai_edge_torch/generative/examples/stable_diffusion/convert_to_tflite.py:52]: what's the token length used by BYOM? is it static as well?

File: ai_edge_torch/generative/examples/stable_diffusion/convert_to_tflite.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  context = torch.cat([context_cond, context_uncond], axis=0)
+  time_embedding = util.get_time_embedding(timestamp)
+
+  # CLIP text encoder
+  ai_edge_torch.signature('encode', clip, (prompt_tokens,)).convert().export(
+      '/tmp/stable_diffusion/clip.tflite'
+  )
+
+  # Image encoder
+  ai_edge_torch.signature('encode', encoder, (input_image, noise)).convert().export(
```
Comment (haozha111) [inline ai_edge_torch/generative/examples/stable_diffusion/convert_to_tflite.py:73]: can you add a TODO to convert to a multi-sig tflite model in the future?

Comment (yichunk) [reply ai_edge_torch/generative/examples/stable_diffusion/convert_to_tflite.py:74]: Done

**Comment by haozha111** [comment]
can you also add the bug number to PR description?

**Review by haozha111 — APPROVED** [review]

**Pull request merged**